index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>SD Global Conference</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
```

Comment:
- [ ] The deployed live version of the home and about page doesn't resemble the given design. Please check the live link and fix it. It has to do with an error in loading some asset files.

<img src='https://user-images.githubusercontent.com/33323077/129225818-e98718d2-0546-4210-887d-22beb4d0a28c.png' height='250px'>